src/main/java/org/alfresco/repo/security/sync/ldap/LDAPUserRegistry.java

```diff
@@ -882,6 +883,26 @@ else if (hasAttributeValue(objectClass, LDAPUserRegistry.this.groupType))
                                             continue;
                                         }
                                     }
+                                    catch (ServiceUnavailableException e)

```

Comment:
Both exceptions have the same outcome, please join them in one catch.

Reply:
Sure.  Thanks.